src/main/host/status_listener.c

```diff
@@ -29,22 +30,28 @@ struct _StatusListener {
     /* The function we call to free the callback argument. */
     StatusArgumentFreeFunc argumentFreeFunc;
 
+    /* Enables deterministic sorting of listener items. */
+    uint64_t deterministicSequenceValue;
+
     /* Memory accounting. */
     int referenceCount;
     MAGIC_DECLARE;
 };
 
 StatusListener* statuslistener_new(StatusCallbackFunc notifyFunc, void* callbackObject,
                                    StatusObjectFreeFunc objectFreeFunc, void* callbackArgument,
-                                   StatusArgumentFreeFunc argumentFreeFunc) {
+                                   StatusArgumentFreeFunc argumentFreeFunc, Host* hostp) {
+    Host* host = (Host*)hostp;
```

Comment:
What's this line for?

Reply:
Oops, that's from an intermediate hack and should be removed :) Good catch!